app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -302,6 +305,6 @@ def pod_definition
   end
 
   def inspector_image
-    'docker.io/openshift/image-inspector:v1.0.z'
+    'docker.io/efreiber/image-inspector:latest'
```

Comment:
No. That's something that wasn't supposed to be pushed.

However, we'll require a new image, probably: `docker.io/openshift/image-inspector:v1.1.z`